packages/core/useDraggable/demo.vue

```diff
@@ -53,5 +53,26 @@ const { x, y, style } = useDraggable(el, {
         {{ Math.round(x) }}, {{ Math.round(y) }}
       </div>
     </Draggable>
+
+    <Draggable
+      v-slot="{ x, y }"
+      p="x-4 y-2"
+      border="~ gray-400/30 rounded"
+      shadow="~ hover:lg"
+      class="fixed bg-$vp-c-bg select-none z-10"
+      :initial-value="{ x: innerWidth / 3.9, y: 240 }"
+      :prevent-default="true"
+      handle=".handle"
```

Comment:
I am not so sure about using a selector string here. I would prefer always to accept element instances using template ref.

Reply:
I have no opinion. I will romove it, please check the changes.